ozaria/site/components/play/PageIntroLevel/index.vue

```diff
@@ -69,163 +61,137 @@
       }
       await this.loadIntroLevel()
     },
-    methods: Object.assign(
-      {},
-      mapActions({
-        fetchCampaign: 'campaigns/fetch'
-      }),
-      {
-        loadIntroLevel: async function () {
-          this.dataLoaded = false
-
-          // Reading query params because this is rendered via backbone router and cannot be directly passed in as props
-          // They need to be in a specific order in the url to read and send them as props directy from backbone router, hence using query params here.
-          this.courseInstanceId = this.courseInstanceId || utils.getQueryVariable('course-instance')
-          this.codeLanguage = this.codeLanguage || utils.getQueryVariable('code-language')
-          this.courseId = this.courseId || utils.getQueryVariable('course')
-          this.campaignId = this.campaignId || utils.getQueryVariable('campaign')
-          try {
-            this.introLevelData = await api.levels.getByIdOrSlug(this.introLevelIdOrSlug)
-            if (me.isSessionless()) { // not saving progress/session for teachers
-              this.language = this.codeLanguage || defaultCodeLanguage
-            } else {
-              this.introLevelSession = await api.levels.upsertSession(this.introLevelIdOrSlug, { courseInstanceId: this.courseInstanceId })
-              this.language = this.introLevelSession.codeLanguage
-            }
-
-            this.introContent = this.introLevelData.introContent
+    methods: {
+      loadIntroLevel: async function () {
+        this.dataLoaded = false
 
-            // fetch campaign data
-            await this.loadCampaignData()
-          } catch (err) {
-            console.error('Error in creating data for intro level', err)
-            // TODO handle_error_ozaria
-            noty({ text: 'Error in creating data for intro level', type: 'error', timeout: 2000 })
-            return
+        // Reading query params because this is rendered via backbone router and cannot be directly passed in as props
+        // They need to be in a specific order in the url to read and send them as props directy from backbone router, hence using query params here.
+        this.courseInstanceId = this.courseInstanceId || utils.getQueryVariable('course-instance')
+        this.codeLanguage = this.codeLanguage || utils.getQueryVariable('code-language')
```

Comment:
Here we are checking the query variable `code-language`.

The naming switches between `code-language` and `codeLanguage`. Should only one be used?

Reply:
great pickup, will update